ddtrace/llmobs/_integrations/openai.py

```diff
@@ -108,6 +108,11 @@ def record_usage(self, span: Span, usage: Dict[str, Any]) -> None:
             if not num_tokens:
                 continue
             span.set_metric("openai.response.usage.%s_tokens" % token_type, num_tokens)
+        for token_type in ("input", "output", "total"):
```

Comment:
Instead of duplicating the same logic let's just add `input/output` into line 106:
```python
for token_type in ("prompt", "completion", "input", "output", "total"):
```